### PR TITLE
feat: support macos log

### PR DESCRIPTION
**Pull request by wxitcode**
## Changes

Support logging on macOS in the same way as iOS.

Logs can be viewed through Instruments and Console.

## Checklist

- [ ] An issue to be fixed by this PR is listed above.
- [ ] New tests are added to ensure new features are working. Please refer to [this page](https://cjycode.com/flutter_rust_bridge/guides/contributing/tip) to see how to add a test.
- [ ] `./frb_internal precommit --mode slow` (or `fast`) is run (it internal runs code generator, does auto formatting, etc).
- [ ] If this PR adds/changes features, documentations (in the `./website` folder) are updated.
- [ ] CI is passing. Please refer to [this page](https://cjycode.com/flutter_rust_bridge/guides/contributing/tip) to see how to solve a failed CI.

## Remark for PR creator

- `./frb_internal --help` shows utilities for development.
- If fzyzcjy does not reply for a few days, maybe he just did not see it, so please ping him.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ keywords = ["flutter", "dart", "ffi", "code-generation", "bindings"]
 categories = ["development-tools::ffi"]
 
 [workspace.dependencies]
-allo-isolate = { version = "0.1.25", features = ["anyhow"] }
+allo-isolate = "0.1.25"
```
Comment (fzyzcjy) [inline Cargo.toml:35]: nit: I guess keeping the original would be good

**Comment by fzyzcjy** [comment]
Good job! After fixing CI (maybe applying the suggestion above first and see what is going on) ready to merge it

File: Cargo.toml (hunk 1 above)
Comment (wxitcode) [reply Cargo.toml:35]: `../flutter_rust_bridge-2.2.0/Cargo.toml`

There is two anyhow feature.

``` toml
[target."cfg(not(target_family = \"wasm\"))".dependencies.allo-isolate]
version = "0.1.25"
features = [
    "anyhow",
    "anyhow",
    "backtrace",
    "zero-copy",
]
```

Comment (fzyzcjy) [reply Cargo.toml]: Oh then it is reasonable to remove duplicate

**Comment by wxitcode** [comment]
ci failed just wait for new version rust nightly. https://github.com/rust-lang/cargo/pull/14370 

```
error: "/home/runner/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/Cargo.lock" does not exist, unable to build with the standard library, try:
        rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
```

**Comment by fzyzcjy** [comment]
Sure, then just wait a bit!

**Pull request merged**